desktop-themes/TraditionalOk/gtk-3.0/assets/Makefile.am

```diff
@@ -0,0 +1,133 @@
+THEME_NAME=TraditionalOk
+
+themedir = $(datadir)/themes/$(THEME_NAME)/gtk-3.0/assets
+
+theme_DATA = \
+   bullet-symbolic.svg \ 
```

Comment:
This file outputs the following error on everyone of these lines:
```
desktop-themes/TraditionalOk/gtk-3.0/assets/Makefile.am:6: warning: whitespace following trailing backslash
```
Please remove trailing whitespace